convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -60,6 +60,11 @@ def run(self):
 
         if not system_info.has_internet_access:
             logger.warning("Skipping the check because no internet connection has been detected.")
+            self.add_message(
+                level="WARNING",
+                id="CONVERT2RHEL_LATEST_CHECK_SKIP_NO_INTERNET",
+                message="Skipping the check because no internet connection has been detected.",
+            )
```

Comment:
```suggestion
            message = "Skipping the check because no internet connection has been detected."
            logger.warning(message)
            self.add_message(
                level="WARNING",
                id="CONVERT2RHEL_LATEST_CHECK_SKIP_NO_INTERNET",
                message=message,
            )
```